src/MojiraBot.ts

```diff
@@ -42,7 +43,7 @@ export default class MojiraBot {
 			// Register events.
 			EventRegistry.setClient( this.client );
 			EventRegistry.add( new ErrorEventHandler() );
-			EventRegistry.add( new RemoveReactionEventHandler( this.client.user.id ) );
```

Comment:
Why was that event handler removed?

Reply:
Ah, good catch. Something must have gone wrong during rebasing.